defaults/main.yml

```diff
@@ -74,12 +80,14 @@ omero_server_selfsigned_certificates: true
 omero_server_ice_version: "3.6"
 
 omero_server_python_requirements_ice_package:
+
   RedHat:
-    9: "https://github.com/glencoesoftware/zeroc-ice-py-rhel9-x86_64/releases/download/\
-      20230830/zeroc_ice-3.6.5-cp39-cp39-linux_x86_64.whl"
+    9: "https://github.com/glencoesoftware/zeroc-ice-py-linux-x86_64/releases/download/20240202/zeroc_ice-3.6.5-cp{{ omero_server_python_version }}-cp{{ omero_server_python_version }}-manylinux_2_28_x86_64.whl"
```

Comment:
We need to indicate in the README the range of version supported <= 3.12 (3.14 will be out later this year)